cmd/admin-router.go

```diff
@@ -48,6 +48,8 @@ func registerAdminRouter(router *mux.Router, enableConfigOps, enableIAMOps bool)
 
 	// Info operations
 	adminRouter.Methods(http.MethodGet).Path(adminAPIVersionPrefix + "/info").HandlerFunc(httpTraceAll(adminAPI.ServerInfoHandler))
+	// Admin-Info operations
```

Comment:
I think this is a breaking change. You will have to use the existing `info` handler and not introduce another one